spec/fixtures/client/browser.yml

```diff
@@ -4465,10 +4465,10 @@
     family: Chrome
   headers:
     Sec-CH-UA: '" Not A;Brand";v="99.0.0.0", "Chromium";v="98.0.4758.109", "Gener8";v="98.0.4758.109"'
-    Sec-CH-UA-Platform: "Windows"
+    Sec-CH-UA-Platform: '"Windows"'
```

Comment:
FYI, these kind of changes will be overwritten on the first fixtures sync from the upstream: fixtures are not updated manually. There is `bundle exec rake update_fixtures` specifically for that: https://github.com/podigee/device_detector/blob/e68ee76b1cc55360074b9e3687ab4e49f8956315/Rakefile#L69-L88

I would suggest to add a dedicated test for the `unquote` functionality outside of the existing fixtures. Some examples can be found in here: https://github.com/podigee/device_detector/tree/develop/spec/device_detector